dbt_subprojects/dex/models/trades/test/_schema.yml

```diff
@@ -0,0 +1,67 @@
+version: 2
+
+models:
+  - name: dex_ci_trades_test
+    meta:
+      blockchain: arbitrum, avalanche_c, base, bnb, celo, ethereum, fantom, gnosis, kaia, optimism, polygon, scroll, zksync, linea, blast, sei, ronin
+      sector: dex
+      contributors: hosuke
+    config:
+      tags: ['dex', 'trades', 'ci_test']
+    description: "CI test view that processes modified base_trades models to verify their integration with the dex.trades pipeline"
+    data_tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - blockchain
+            - project
+            - version
+            - tx_hash
+            - evt_index
```

Comment:
i think we're safe to remove this, as we apply to every project level spell. probably overkill?
i guess if performance of this test is fast, it doens't hurt

Reply:
I think we can remove this, it takes the same amount of time as usd value check:
https://github.com/duneanalytics/spellbook/actions/runs/12733136697/job/35489130037?pr=7429#step:15:39

Reply:
```suggestion
```